src/backend/Level0/CHIPBackendLevel0.cc

```diff
@@ -476,7 +454,7 @@ bool CHIPEventLevel0::updateFinishStatus(bool ThrowErrorIfNotReady) {
 /** This Doesn't work right now due to Level Zero Backend hanging?
   unsinged long CHIPEventLevel0::getFinishTime() {
 
-    ze_kernel_timestamp_result_t res{};
+    ze_kernel_Timestampresult_t res{};
```

Comment:
Is this change intended? Seems accidental.

Reply:
Indeed!

Reply:
I don't see this  in the actual code? This should cause a build failure